.travis.yml

```diff
@@ -22,6 +22,10 @@ env:
     - CI_NODE_INDEX=3
     - CI_NODE_INDEX=4 KARMA="true" GITHUB_DEPLOY="true"
 
+before_install:
+  - gem install bundler
```

Comment:
I'm not convinced. If we have different execution results because of an old bundler version, then I would like to know that before we deploy to production. Or do you think that it would affect Travis, but not a production server?

If we want to talk about updating bundler on the production servers, that's another topic, right. But that topic needs to be discussed before we merge this, I think. A discussion about unifying the bundler version would be really good, actually.

What was the motivation for this change? Did you have problems with bundler?

By the way, I wouldn't use my code from above just like that. Not all bundler versions/configuration append their version. And I'm not sure about the whitespace at the beginning of that line.